assignment_3.py

```diff
@@ -0,0 +1,120 @@
+def Psyco(input_student):
+    return input_student
+print(Psyco("pavi")) #pavi
+def ds(input_students):
+    return input_students + 15
+print(ds(15))  #25
+def ds(input_students):
+    return input_students + 'string'
+print(ds("pavi")) #pavistring
+def ds(input_students):
+    return input_students + 10
+try:
+    ds("pavi")
+except:
+    print("string+int are not concadenate") #string+int are not concadenate
+''' ouput
+pavi
+20
+pavistring
+string+int are not concadenate
+'''
+
+class Calculation:

```

Comment:
I have mentioned multiple times. please do not come up with Calculator function again. come up with story like class and functions.